src/raw/text-bold/icon_16.svg

```diff
@@ -0,0 +1,10 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="none" viewBox="0 0 16 16">
+    <g clip-path="url(#a)">
```

Comment:
clipPaths and defs should be removed. Applies to all text-underline, text-bold, text-italic and text-strikethrough icons in this PR.